src/domains/shielder/stores/getShielderIndexedDB.ts

```diff
@@ -156,8 +156,9 @@ export const getTransactionsIndexedDB = (accountAddress: string) => {
 
 export const clearShielderIndexedDB = async () => {
   const db = await openDB(DB_NAME, DB_VERSION);
-  await Promise.all([
-    db.clear(STORE_CLIENTS),
-    db.clear(STORE_TRANSACTIONS),
-  ]);
+
+  await Promise.all(
+    [STORE_CLIENTS, STORE_TRANSACTIONS]
+      .map(store => typeof db.clear === 'function' ? db.clear(store) : undefined)
+  );
```

Comment:
Is the function check really necessary? Looks like it works form me without it:
```suggestion
  await Promise.all([
    STORE_CLIENTS,
    STORE_TRANSACTIONS,
  ].map(db.clear));
```

Reply:
Without it chromatic build fails :(

Reply:
Okay, in general we shouldn't adapt source code to tests unless its in favour of the source code itself (e.g. the structure gets cleaner, or dependencies are injected etc.).

So would be nice to get rid of this. Although of course if you've spent considerate time on it and it appears to be a bigger problem, than we can have an exception.